include/motor-control/core/stepper_motor/motor_interrupt_handler.hpp

```diff
@@ -110,11 +114,13 @@ class MotorInterruptHandler {
          * This function is called from a timer interrupt. See
          * `motor_hardware.cpp`.
          */
-        if (!has_active_move && has_messages()) {
+        if (!has_active_move && has_move_messages()) {
             update_move();
+            handle_update_position_queue_error();
```

Comment:
how much time does this check add to the work done in the isr?

Reply:
I don't have a quantitative answer, but this is a relatively low-overhead thing to add in because it doesn't have any hardware interaction, most of the time is interacting with FreeRTOS queues. Tested running a home command and then sending an UpdateMotorPositionRequest during the movement on the Head board with both interrupts active and the result was as expected, an error message came back and the motor interrupt kept running as usual.